samples/DataSourceWithProcessingAndExtensions/Sample/MetadataTables/EventStatsTable.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Performance.SDK.Extensibility;
+using Microsoft.Performance.SDK.Processing;
+using DataExtensionsSample.SourceDataCookers;
+
+namespace DataExtensionsSample.MetadataTables
+{
+    [Table]
+    [RequiresCooker(EventStatsCooker.CookerPathAsString)]
+    public class EventStatsTable
+    {
+        public static TableDescriptor TableDescriptor = new TableDescriptor(
+            Guid.Parse("E6B3FE19-7E1F-456C-AFC8-25A998B01F53"),
+            "Event Stats",
+            "Event Stats",
+            TableDescriptor.DefaultCategory,
+            true);
+
+        private static readonly ColumnConfiguration EventNameConfiguration = new ColumnConfiguration(
+            new ColumnMetadata(new Guid("{690910AF-0451-40AC-BF5C-15B75B326795}"), "Event Name", "Event Name"),
```

Comment:
>"Event Name" [](start = 97, length = 12)

This is the only ColumnMetadata with a description. Can we change this something more descriptive?